src/main/java/com/akademia/educators_back/controller/ProblemController.java

```diff
@@ -22,11 +25,21 @@ public class ProblemController {
     private static final Logger LOG = LoggerFactory.getLogger(ProblemController.class);
     public ProblemServiceImpl problemService;
 
+    /**
+     * Controller get to problem list.
```

Comment:
- GET z dużych liter
- lepiej by brzmiało coś w deseń "Controller responsible for sending list of all Problem objects" albo "Controller handling request for retreiving list of Problem objects and sending them to client", tak że jak wpiszesz np w translator to dobrze przetłumaczy
- polecam DeepL jako tranlator ;)